man/man1/registercloudguest.1

```diff
@@ -0,0 +1,58 @@
+.\" Process this file with
+.\" groff -man -Tascii registercloudguest.1
+.\"
+.TH registercloudguest.1
+.SH NAME
+registercloudguest \- Register a SUSE Linux Enterprise on-demand instance
+.SH SYNOPSIS
+.B registercloudguest [option]
+.SH DESCRIPTION
+.B registercloudguest
+Register a SUSE Linux Enterprise on-demand instance with the region local
```

Comment:
The infrastructure is _not always_ region local... especially in Azure where they have explicitly asked that we not put infrastructure in every region.